vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -229,15 +230,7 @@ testKubeBinariesPresent() {
   echo "$test:Start"
   containerRuntime=$1
   binaryDir=/usr/local/bin
-  k8sVersions="
-  1.22.11-hotfix.20220620
-  1.22.15
-  1.23.8-hotfix.20220620
-  1.23.12
-  1.24.3
-  1.24.6
-  1.25.2-hotfix.20221006
-  "
+  k8sVersions="$(jq -r .kubernetes.versions[] --monochrome-output --compact-output < /opt/azure/manifest.json)"
```

Comment:
is --compact-output necessary here? why not in install-deps if so?

Reply:
It's not necessary. It was used in a couple of other places and I added it while debugging another issue. Can remove it.